tests/test.cpp

```diff
@@ -404,7 +404,7 @@ static int TestCrtReportHook(int reportType, char *message, int *)
 int main(int argc, char **argv)
 {
     // tests can be ran non-interactively so make sure we don't show any assert
-    // dialog boxes -- neither our own nor from MSVC debug CRT -- which would
+    // dialog boxes -- either our own nor from MSVC debug CRT -- which would
```

Comment:
I'm almost sure this is incorrect, "either ... nor" doesn't really work, does it?

Reply:
Yes, this one is correct as-is.

Reply:
Agreed, there was an error here, and the change has been undone. Please pick another two random changes.